### DISCUSSION TITLE
Where put the name prefixes like Sr. Sra. Dr. Phd?

### DISCUSSION THREAD

**Discussion by webysther** [monicahq/monica#7300] [Q&A] [answered]
I have my doctors contacts and will be nice to put the prefix.

https://en.wikipedia.org/wiki/English_honorifics

**Comment by djaiss** [comment, marked as answer]
When you edit a contact, you have those two fields

<img width="491" alt="image" src="https://github.com/monicahq/monica/assets/61099/30de1e51-46ea-4286-aac8-fdbd7c264cf8">

With this, you can write basically anything you want.

**Reply by webysther** [reply]
Thank you! This is only in the version 5?
